### DISCUSSION TITLE
Why can't I use color in textStyles? Actually it has the wrong type

### DISCUSSION THREAD

**Discussion by topkat** [chakra-ui/panda#2619] [Q&A]
### Description

<img width="773" alt="image" src="https://github.com/chakra-ui/panda/assets/17770809/4ebc442f-81ef-4c0f-bb3f-4603d19c5030">

Actually it seems happy with a nested field but the problem keeps persisting on a higher level:

<img width="301" alt="image" src="https://github.com/chakra-ui/panda/assets/17770809/e3927229-4582-4583-b672-3f7f4335b14a">

### Link to Reproduction

https://play.panda-css.com/lopuCmaA5t

### Steps to reproduce

Minimal example:

``` typescript
import { defineTextStyles } from '@pandacss/dev'

export const textStyles = defineTextStyles({
  emphasize: {
    description: 'coucou azdjkn ndzjaod jh ',
    value: {
      color: 'green',
    }
  }
})
```

### Panda CSS Version

^0.39.1

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment]
My perspective on this is that color is very context-specific and should not belong in textStyles. It's usually best to define color in the page or component where it needs to be used.

Design tools like Figma and Sketch also recommend this, so I'm leaning toward keeping it as is.

**Reply by topkat** [reply]
While I agree with you, I think that in general the whole point of best practices is that they should not be enforced because in certain context you need to set a default color in text style without frictions. 
Also, while this example is with a solid color, we can't either use it with a color token. Is it a bad practice to have a default color token for text in an application ?
